### ISSUE TITLE
Enhancement: remember TOC file line position when returning to TOC

### ISSUE THREAD

**Issue by srustamo** [wasamasa/nov.el#19]
Currently hitting <kbd>t</kbd> when in the main text of an EPUB brings up TOC with the point at the first char. Scrolling to, say, line 10 of the TOC, hitting <kbd>Enter</kbd>, opens whatever chapter line 10 was referring. Hitting <kbd>t</kbd> again opens TOC at first char of the file again, instead of the previous position at line 10, a bit disorienting if the TOC is large.

**Comment by wasamasa** [comment]
Why special-case the TOC here? Wouldn't it make sense to remember the document position for each document in the EPUB spine?  Granted, I'd imagine that you'd need to make adjustments like only `t`, `n` and `p` going to the last remembered position whereas `SPC` and `S-SPC` wouldn't, but other than that I don't see the problem.

**Comment by srustamo** [comment]
Agree, <kbd>n</kbd>, <kbd>p</kbd> could also benefit from remembering their last position. By the way, why use <kbd>SPC</kbd> for pagination instead of <kbd>j</kbd>, <kbd>k</kbd>? Perhaps just add them as defaults, too, since they are not bound in this major mode.

**Comment by srustamo** [comment]
I thought a bit more about TOC location: perhaps it is more useful to be able to go the chapter entry in the TOC from the document the <kbd>t</kbd> was pressed in?

**Comment by wasamasa** [comment]
This is Emacs, not Vim, so `j` and `k` won't make any sense for the target group.  Feel free to bind them yourself, you heretic.

If `t` was being this smart, then yes, it would make much more sense to special-case it this way.  The proposed behavior for `n` and `p` would make for their own ticket then.

**Comment by j3ky** [comment]
> you heretic 

hahahaha
